SwiftPackageManager.md

```diff
@@ -44,6 +44,10 @@ Choose the Firebase products that you want installed in your app.
 If you've installed FirebaseAnalytics, Add the `-ObjC` option to `Other Linker Flags`
 in the `Build Settings` tab.
 
+If you're using FirebaseAnalytics, the Xcode 12 GM, and have an issue with
```

Comment:
Nit: s/the Xcode 12 GM/Xcode 12.0/

Not sure if it repros with 12.2 yet though

Reply:
Fixed.

According to https://github.com/firebase/firebase-ios-sdk/issues/6472#issuecomment-694460700, it's in the 12.2 beta still.